### PR TITLE
feat: add dev generation support via !seeddev

### PR DESCRIPTION
**Pull request by TreZc0**
also changing zsr domains to ootr for consistency.

### CODE REVIEW

File: randobot/handler.py [hunk 1]
```diff
@@ -87,14 +87,14 @@ async def begin(self):
         self.heartbeat_task = create_task(self.heartbeat(), name=f'heartbeat for {self.data.get("name")}')
         if not self.state.get('intro_sent') and not self._race_in_progress():
             await self.send_message(
-                'Welcome to OoTR! Create a seed with !seed <preset>'
+                'Welcome to OoTR! Create a release seed with !seed <preset> or a latest dev seed with !seeddev <preset>'
             )
             await self.send_message(
                 'If no preset is selected, weekly settings will be used. '
-                'Use !spoilerseed to generate a seed with a spoiler log.'
+                'Use !race to generate a release seed with a spoiler log.'
```
Comment (fenhl) [inline randobot/handler.py:94]: Looks like `!race` hasn't actually been implemented?

Comment (TreZc0) [reply randobot/handler.py:94]: Hm I just made it consistent cause line 194 already called it !race before, but probably changed it in the wrong place lol.

Comment (deains) [reply randobot/handler.py:94]: Think that's a holdover from an old version. It was just in a code comment.

File: randobot/handler.py [hunk 2]
```diff
@@ -169,23 +169,39 @@ async def ex_seed(self, args, message):
         """
         if self._race_in_progress():
             return
-        await self.roll_and_send(args, message, True)
+        await self.roll_and_send(args, message, True, False)
```
Comment (deains) [inline randobot/handler.py]: Can we use keyword args for the booleans on these calls as it's a bit hard to read, i.e. `self.roll_and_send(args, message, encrypt=True, dev=False)`

File: randobot/handler.py [hunk 3]
```diff
@@ -327,13 +344,17 @@ async def load_seed_hash(self):
             'seed_url': self.seed_url % self.state['seed_id'],
         })
 
-    async def send_presets(self):
+    async def send_presets(self,dev):
```
Comment (deains) [inline randobot/handler.py]: Whitespace

**Review by deains — CHANGES_REQUESTED** [review]

**Comment by TreZc0** [comment]
Alright, have resolved those.

**Pull request merged**